diff -urN xmonad-contrib-0.11.4.orig/XMonad/Hooks/DebugEvents.hs xmonad-contrib-0.11.4/XMonad/Hooks/DebugEvents.hs
--- xmonad-contrib-0.11.4.orig/XMonad/Hooks/DebugEvents.hs	2015-03-29 20:31:49.750052992 +0200
+++ xmonad-contrib-0.11.4/XMonad/Hooks/DebugEvents.hs	2015-03-29 20:33:37.373383041 +0200
@@ -30,6 +30,7 @@
 
 -- import           Graphics.X11.Xlib.Extras.GetAtomName        (getAtomName)
 
+import           Control.Applicative
 import           Control.Exception.Extensible         as E
 import           Control.Monad.State
 import           Control.Monad.Reader
diff -urN xmonad-contrib-0.11.4.orig/XMonad/Util/Invisible.hs xmonad-contrib-0.11.4/XMonad/Util/Invisible.hs
--- xmonad-contrib-0.11.4.orig/XMonad/Util/Invisible.hs	2015-03-29 20:32:10.070052372 +0200
+++ xmonad-contrib-0.11.4/XMonad/Util/Invisible.hs	2015-03-29 20:31:03.553387735 +0200
@@ -22,6 +22,8 @@
                             , fromIMaybe
                             ) where
 
+import Control.Applicative
+
 -- $usage
 -- A wrapper data type to store layout state that shouldn't be persisted across
 -- restarts. A common wrapped type to use is @Maybe a@.
